### ISSUE TITLE
problem loading PropertyTools.Wpf with released app

### ISSUE THREAD

**Issue by FrankCAD** [PropertyTools/PropertyTools#239]
For use in Autodesk AutoCAD 2018 and higher I'm developing a plug-in. The plug-in is a set of dll assemblies developed in C# and VB. In one of my assemblies I'm using (trying to) the PropertyGrid control. Debugging during development in VS2017 everything works fine. But using the released and installed assembly-set, when trying to open a window with the PropertyGrid control, an error is displayed stating the assembly PropertyTools.Wpf cannot be found. See attached image. All assembly files of the plug-in are combined in one directory and PropertyTools.Wpf is the only assembly that can't found.
I am not an expert in this area, so can someone help me to solve this problem.

![Error PropertyTools Wpf](https://user-images.githubusercontent.com/62705627/77655868-51a65980-6f73-11ea-8f16-f327653daace.PNG)

**Comment by objorke** [comment]
could it be a signing issue? are the other assemblies signed?

**Comment by FrankCAD** [comment]
No. The assemblies I created are not signed. Until now i never had to do with signing.

**Comment by FrankCAD** [comment]
The error message attached to the initial comment was created from a build where the source PropertyTools.Wpf was integrated in my project. Maybe that message gave a improper impression.
The error message attached to this message is created from a build where the reference to PropertyTools.Wpf is from the NuGet package. Maybe this leads to a better understanding of the problem.
Adding binding redirect configuration in the App.config did solve the issue.
I still have no clue how to solve this problem.

![Error2 PropertyTools Wpf](https://user-images.githubusercontent.com/62705627/78376109-22749580-75ce-11ea-974f-d0c5389e8fdd.PNG)
